Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -152,6 +158,18 @@ private void StartAndAttachDebugger(string file, string nodePath)
             dbgInfo.bstrExe = $"\01";
 
             AttachDebugger(dbgInfo);
+
+            if (startBrowser && uri != null)
+            {
+                OnPortOpenedHandler.CreateHandler(
+                    uri.Port,
+                    shortCircuitPredicate: () => process.HasExited,
+                    action: () =>
+                    {
+                        VsShellUtilities.OpenBrowser(url, (uint)__VSOSPFLAGS.OSP_LaunchNewBrowser);
```

Comment:
Yeah, I think this is the VS API to launch the OS default browser.... but I think that's the same behavior as previous versions of Node. I'll investigate how to grab the selected browser, though that may require interaction with some ASP.NET code.